### ISSUE TITLE
[2.6.6] Remove RKE1 Windows provisioning due to EOL

### ISSUE THREAD

**Issue by rosskirkpat** [rancher/terraform-provider-rke#343]
As part of the RKE1 Windows EOL upcoming in Rancher v2.6.6, we need to remove the ability to provision new RKE1 Windows clusters in the rke terraform provider.

https://github.com/rancher/windows/issues/179
https://github.com/rancher/windows/issues/182

**Comment by paynejacob** [comment]
@rosskirkpat 2.6.6 is scheduled to be released before September do we want to disable this functionality before then or wait for a later release?

**Comment by rosskirkpat** [comment]
@paynejacob Yes, the functionality needs to be deprecated and include a warning when someone tries to use the resource(s) but not removed.
